WalletWasabi.Fluent/Styles/Button.axaml

```diff
@@ -1,5 +1,23 @@
 <Styles xmlns="https://github.com/avaloniaui"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+  <Styles.Resources>
+    <Thickness x:Key="ActionButtonPadding">20,10</Thickness>
+    <Thickness x:Key="InvisibleButtonPadding">20,10</Thickness>
+
+    <ControlTheme x:Key="AccentButton" TargetType="Button" BasedOn="{StaticResource {x:Type Button}}">
+      <Setter Property="Background" Value="{DynamicResource SystemAccentColor}" />
+      <Setter Property="CornerRadius" Value="6" />
+      <Setter Property="Foreground" Value="{DynamicResource AcrylicTrimForeground}" />
+      <Setter Property="Padding" Value="9 7" />
+      <Style Selector="^:pointerover /template/ ContentPresenter">
+        <Setter Property="Background" Value="{DynamicResource FadedAccentBrush}" />
+        <Setter Property="Foreground" Value="{DynamicResource AcrylicTrimForeground}" />
+      </Style>
```

Comment:
Pressed style also needed,
That should be a darker shade of accent color

Reply:
Done.